src/main/scala/chisel3/util/experimental/BoringUtils.scala

```diff
@@ -263,11 +265,35 @@ object BoringUtils {
     val (upPath, downPath) = lcaResult.get
     val lcaSource = drill(source, upPath.tail, true)
 
-    /** Creating a wire to assign the result to.  We will return this. */
-    val bore = Wire(purePortType)
-    val sink = drill(bore, downPath.tail, false)
+    if (createProbe) {
+      drill(lcaSource, downPath.tail, false)
```

Comment:
@seldridge does this have the same up-down bug that you were trying to fix? I don't want to create the `bore` wire here, so not sure exactly what I should be drilling here.